packages/atomic/src/components/search/atomic-tab-manager/atomic-tab-manager.new.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import type {Meta, StoryObj as Story} from '@storybook/web-components-vite';
+import {getStorybookHelpers} from '@wc-toolkit/storybook-helpers';
+import {html} from 'lit';
+import {parameters} from '@/storybook-utils/common/common-meta-parameters';
+import {wrapInSearchInterface} from '@/storybook-utils/search/search-interface-wrapper';
+
+const {decorator, play} = wrapInSearchInterface();
```

Comment:
Either skip the initial request or mock requests with MSW 😅 

I'd recommend the latter, so if we every decide that we want to do something with the response it'll already be mocked. More future-proof.